.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,16 +1,20 @@
-#### What does this PR do?
-#### Any background context you want to provide?
-#### What ticket does this PR close?
-Connected to [relevant GitHub issues, eg #76]
-#### Where should the reviewer start?
-#### How should this be manually tested?
-#### Screenshots (if appropriate)
-#### Has the Version and Changelog been updated?
-#### Questions:
-> Does this work have automated integration and unit tests?
+### What does this PR do?
+- _What's changed? Why were these changes made?_
+- _How should the reviewer approach this PR, especially if manual tests are required?_
+- _Are there relevant screenshots you can add to the PR description?_
 
-> Can we make a blog post, video, or animated GIF of this?
+### What ticket does this PR close?
+Connected to #[relevant GitHub issues, eg 76]
 
-> Has this change been documented (Readme, docs, etc.)?
+### Checklists
 
-> Does the knowledge base need an update?
+#### Change log
+- [ ] The CHANGELOG has been updated, or
+- [ ] This PR does not include user-facing changes and doesn't require a CHANGELOG update
+
+#### Test coverage
+- [ ] This PR includes new unit or integration tests to go with the code changes
```

Comment:
I have a strong preference towards Unit tests in most cases.  It feels like our integration test first approach has hindered our ability to get fast tests locally.  What do you think about changing `or` to `and`?
```suggestion
- [ ] This PR includes new unit and integration tests to go with the code changes
```